spec/pub_sub_identifier_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe PubSub do
```

Comment:
Due to the way `config` works, I believe each different config setting needs it's own spec file.